data/is_ocaml_yet/web.md

```diff
@@ -0,0 +1,278 @@
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - zlib
+    - decompress
+    - camlzip
+    - lz4
+    - ezgzip
+    - hpack
+    - brotli
+    - zstd
+    - snappy
+    - lzo
+    - zstandard
+    - carton
+  - name: Crypto
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - cryptokit
+    - mirage-crypto
+    - nocrypto
+    - pkcs11
+    - tls
+    - x509
+    - fiat-p256
+    - sha
+    - blake2
+    - blake3
+    - bls12-381
+    - pbkdf
+    - dirsp-proscript
+    - digestif
+  - name: Database
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - caqti
+    - pgocaml
+    - sqlite3
+    - lmdb
+    - irmin
+    - mysql
+    - postgresql
+    - aws-rds
+    - sequoia
+    - sqlgg
+    - petrol
+    - mariadb
+  - name: Email
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - tidy_email
+    - emile
+    - smtp
+    - letters
+    - mrmime
+    - sendmail
+    - received
+    - email_message
+    - colombe
+    - dkim
+    - uspf
+  - name: En- & Decoding
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - tar
+    - imagelib
+    - base64
+    - multipart-form-data
+    - multipart_form
+    - biniou
+  - name: Web Frameworks
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - dream
+    - eliom
+    - opium
+    - sihl
+  - name: HTTP Clients
+    status: 🔴
+    description: |
+      n/a
+    packages:
+    - cohttp
+    - hyper
+    - ocurl
+    - http-lwt-client
+    - ezcurl
+  - name: Internationalization
+    status: 🆘
+    description: |
+      n/a
+    packages:
+    - gettext
+  - name: Logging
+    status: 🟠
+    description: |
+      n/a
+    packages:
+    - logs
+    - dolog
+    - tracing
+    - catapult
+    - opentelemetry
+    - prometheus
+  - name: Lower Web-Stack
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - httpaf
+    - http
+    - tls
+    - websocket
+    - websocketaf
+    - gluten
+    - awa
+    - uri
+    - dns
+    - tcpip
+    - charrua
+    - mirage-nat
+    - arp
+    - ethernet
+    - paf
+    - mimic
+    - tiny_httpd
+  - name: Serializers
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - atd
+    - yojson
+    - jsonm
+    - sexplib
+    - yaml
+    - csv
+    - bencode
+    - toml
+    - xml-light
+    - graphql
+    - omd
+    - cmarkit
+    - css
+    - csexp
+    - data-encoding
+  - name: External Services
+    status: 🆘
+    description: |
+      n/a
+    packages:
+    - amqp-client
+    - kafka
+    - redis
+    - zmq
+  - name: Syndication/RSS
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - syndic
+    - river
+    - rss
+  - name: Templating
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - embedded_ocaml_templates
+    - mustache
+    - jingoo
+  - name: Static Site Generation
+    status: 🟡
+    description: |
+      n/a
+    packages:
+    - finch
+    - stone
+    - camyll
+    - dream-serve
+    - soupault
```

Comment:
[`yocaml`](https://github.com/xhtmlboi/yocaml) is another solution used by many too.

Reply:
Yes but ... not available on OPAM... :(

Reply:
@xhtmlboi would you maybe be open to having yocaml published as an opam package?

Reply:
I'm really sorry but I've never really used OPAM so I don't have no idea how to publish a package on OPAM.

Reply:
@xhtmlboi would you be okay with it if someone else stepped up to help? I'm sure we'll find someone willing to put some effort to make it happen.

Reply:
In order to be `<= 5.0.x`, I'm sure that I'll probably make some changes (for example, rename the module `Effect`). 
When it is done, I'll try to publish the repo on OPAM and if this branch is merged, I'll make a pull request. Thanks for the suggestion. Since @dinosaure is a power user of maintening OPAM packages, I guess that I can ask him for help if I struggle too much.

Reply:
I added a link to the GitHub repo. :)